src/components/Loader/peopleTable.tsx

```diff
@@ -0,0 +1,236 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from './Loader';
+
+export const PeopleTable: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const { personId } = useParams();
+  const selectedPerson = people.find(person => person.slug === personId);
+  const [isPeopleLoaded, setIsPeopleLoaded] = useState(true);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+
+  const loadPeople = async () => {
+    setIsPeopleLoading(true);
+
+    try {
+      const peopleData = await getPeople();
+
+      setPeople(peopleData);
+    } catch {
+      setIsPeopleLoaded(false);
+    } finally {
+      setIsPeopleLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  const isMotherExistInList = (motherName?: string | null) => {
+    return people.find(person => person.name === motherName);
+  };
+
+  const isFatherExistInList = (fatherName?: string | null) => {
+    return people.find(person => person.name === fatherName);
+  };
+
+  const getParent = (person: Person, parentType: 'father' | 'mother') => {
+    const parentTypeName = parentType === 'mother'
+      ? 'motherName'
+      : 'fatherName';
+
+    if (!person[parentTypeName]) {
+      return <td>-</td>;
```

Comment:
`<td>` returns in all cases, better return a value and call this function inside `<td>`.